logging/monolog_console.rst

```diff
@@ -38,19 +38,19 @@ The example above could then be rewritten as::
     namespace App\Command;
```

Comment:
L16 can also be updated to replace execute method by invoke ?

`protected function execute(InputInterface $input, OutputInterface $output): int`